packages/experimental/Button/src/Button.types.ts

```diff
@@ -84,10 +84,7 @@ export interface ButtonCoreProps extends Omit<IWithPressableOptions<ViewProps>,
    */
   icon?: IconSourcesType;
 
-  /**
-   * A RefObject to access the IButton interface. Use this to access the public methods and properties of the component.
-   */
-  componentRef?: React.RefObject<IFocusable>;
+  innerRef: React.ForwardedRef<IFocusable>;
```

Comment:
innerRef is still exposed to users, it seems?

Reply:
Yep good catch.

Reply:
Should be fixed with latest commits